ci/requirements-conversion.txt

```diff
@@ -146,12 +149,12 @@ test-generator==0.1.1
     #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_mxnet.txt
     #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_onnx.txt
     #   -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_tf2.txt
-torch==1.6.0
+torch==1.8.0
```

Comment:
is it strong requirements to use torch 1.8?

Reply:
Yes, with torch version lower 1.8, an runtime error occurs during conversion:
`RuntimeError: Exporting the operator std_mean to ONNX opset version 11 is not supported. Please open a bug to request ONNX export support for the missing operator.`

Reply:
then tools/downloader/requirements-pytorch.in should be also updated

Reply:
Done